### PR TITLE
Add feature ror capstone

### PR DESCRIPTION
**Pull request by yash244466666**
### In this pull request, the following changes have been made:

- [x]  Splash screen - with login and sign-up buttons
- [x]  When the user logs in, the categories page is presented
- [x]  There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
- [x]  user can see the name, icon, and the total amount of all the transactions that belong to that category.
- [x]  When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
- [x]  There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
- [x]  For a given category, the list of transactions is presented, ordered by the most recent.
- [x]  At the top of the page the user can see the total amount for the category
- [x]  When the user clicks on the "Back" button (<), the user navigates to the home page.
- [x]  Add sidebar navigation
- [x]  Add unit and integration tests for the RoR application.
- [x]  Deployed to render

### **Project documentation:** 

- [See the documentation](https://drive.google.com/file/d/17XBHR5j_eB9eGfV9kgHNozvV_kCQHhWT/view?usp=sharing)

### CODE REVIEW

File: app/views/entities/_entity.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <%= entity.name %>
+      </p>
+
+      <p>
+        <strong>Amount:</strong>
+        $<%= entity.amount %>
+      </p>
+    </div>
+<% end %>
+
```
Comment (ichala) [inline app/views/entities/_entity.html.erb:15]: - [ ] Great job so far implementing the general layout of the app based on the design guidelines, but u still missing some important points like :

- Colors
- Typography: font face, size, and weight.
- some composition and space between elements (margins paddings etc...)

**_kindly try to be as close as possible top design guidelines and your project will be ready_**

Your Design           | Guidelines
:-------------------------:|:-------------------------:
image | image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Great use of partials ✔️ 
- App works as the requirements mentioned ✔️ 
- the app is tested ✔️ 


 ## Required Changes ♻️

- [ ] **_🛑 The presentation video that you submitted has no audio ! kindly retake it and make sure that:_**
1. The maximum length of the video is 5 minutes.
2. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
3. The video contains a demo of the project, highlighting all the project features.
4. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yash244466666** [comment]
### Fixed style as instructed:

- [x] Fixed Colors
- [x] Typography: font face, size, and weight.
- [x] some composition and space between elements (margins paddings etc...)

**Comment by prolajumokeoni** [comment]
Hi @yash244466666, Kindly fix the stylelint  errors

**Comment by yash244466666** [comment]
@prolajumokeoni  Sorry for that, linter is fixed

**Comment by prolajumokeoni** [comment]
Hi @yash244466666, you are still  missing the video presentation as required by last reviewer, please see the [video presentation ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#project-documentation) requirements

**Comment by yash244466666** [comment]
> Hi @yash244466666, you are still missing the video presentation as required by last reviewer, please see the [video presentation ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#project-documentation) requirements

@prolajumokeoni sorry that is a mistake. let me fix that, give me 10 minutes

**Comment by yash244466666** [comment]
@prolajumokeoni done adding the video please check https://drive.google.com/file/d/17XBHR5j_eB9eGfV9kgHNozvV_kCQHhWT/view?usp=sharing

**Review by prolajumokeoni — APPROVED** [review]
Hi @yash244466666 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights 
- Styling looks like the Behance design  👍🏿 
- All changes requested have been implemented 🥇  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**